linkml/generators/jsonschemagen.py

```diff
@@ -49,95 +124,71 @@ class JsonSchemaGenerator(Generator):
 
     # ClassVars
     generatorname = os.path.basename(__file__)
-    generatorversion = "0.0.2"
+    generatorversion = "0.0.3"
```

Comment:
we haven't been doing this very consistently, need a strategy here. maybe overall version of linkml is good enough

Reply:
Okay yeah I wasn't sure. I saw it there so I figured I'd increment it. The `gen-json-schema` CLI is already hooked up to use the main LinkML version number when you pass the `--version` flag. I don't see any place the `generatorversion` variable is used within the `linkml` project. I'd be happy to just remove it unless you think there's a possibility of a downstream project using that variable.